web-client/src/presenter/actions/createPractitionerDocumentAction.test.js

```diff
@@ -0,0 +1,52 @@
+import { PRACTITIONER_DOCUMENT_TYPES_MAP } from '../../../../shared/src/business/entities/EntityConstants';
+import { applicationContextForClient as applicationContext } from '../../../../shared/src/business/test/createTestApplicationContext';
+import { createPractitionerDocumentAction } from './createPractitionerDocumentAction';
+import { presenter } from '../presenter-mock';
+import { runAction } from 'cerebral/test';
+
+describe('createPractitionerDocumentAction', () => {
+  beforeAll(() => {
+    presenter.providers.applicationContext = applicationContext;
+  });
+
+  it('should try to upload the file on state to s3', async () => {
```

Comment:
super nitpicky, but i don't think it `to s3` is right here as really these could be getting stored anywhere, and nothing below is really testing that aspect.

Reply:
rephrase to 'should try to upload the file on state to persist the new practitioner document'